static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -30,6 +39,32 @@ export default function FeedbackItem({feedbackItem}: Props) {
   const organization = useOrganization();
   const {projects} = useProjects();
 
+  const api = useApi();
+  const {getParamValue: getFeedbackSlug, setParamValue: setFeedbackSlug} =
+    useUrlParams('feedbackSlug');
+  const {setFeedback} = useInfiniteFeedbackListData();
+  const feedbackId = feedbackItem.feedback_id;
+
+  const url = useMemo(() => {
+    const feedbackSlug = getFeedbackSlug();
+    const projectSlug = feedbackSlug?.split(':')[0];
+    return `/projects/${organization.slug}/${projectSlug}/feedback/${feedbackId}/`;
+  }, [feedbackId, getFeedbackSlug, organization]);
+
+  const handleDelete = useCallback(async () => {
```

Comment:
had to refactor the `deleteButton` file and move the handle here because react hooks were breaking...🤔 maybe there's a better way to do this

Reply:
ya that's sus
the button should've been self contained with only that one prop... hrm

Reply:
oooh, it's because the button is inside the flyout. derp

so it could be resolved with more props passed in from here

Reply:
ohh let me try

Reply:
the main bit is the `handleDelete` callback

could make a hook that just returns that, then pass that into the `<DropdownMenu>`

ie: 

```
const {handeDelete} = useDeleteFeedback({feedbackItem});

return <DropdownMenu
   ....
   items={[
      {
        key: 'delete',
        label: t('Delete'),
        onAction: handeDelete,
       },